src/api/rewrite.ts

```diff
@@ -0,0 +1,10 @@
+import { callSidecar } from './sidecar';
+
+export async function askForRewrite(selection: string): Promise<string> {
+  try {
+    const response = await callSidecar('rewrite', ['--text', String(selection)]);
```

Comment:
The `String()` wrapper here isn't necessary (if the TS types are to be believed!).